.github/workflows/build.yml

```diff
@@ -457,6 +457,9 @@ jobs:
     timeout-minutes: 60
     needs: check_source
     if: needs.check_source.outputs.run_tests == 'true'
+    strategy:
+      matrix:
+        os: [ubuntu-24.04]
```

Comment:
There's still a `runs-on: ubuntu-22.04` a few lines above this.  Maybe that should be changed instead?

Reply:
Yes, just need to replace it with `matrix.os`. BTW, I noticed that the job
`Check if the ABI has changed` is runs on `ubuntu-22.04`, should that be changed to `ubuntu-24.04` too ?